src/assets/sass/protocol/components/_navigation.scss

```diff
@@ -150,16 +150,16 @@
 // Mobile Navigation Icon
 
 .mzp-c-navigation-menu-button {
+    @include bidi(((background-position, right 6px center, left 6px center ),));
     @include bidi(((float, right, left),));
-    @include image-replaced;
-    background: transparent url('#{$image-path}/icons/ui/menu-black.svg') center center no-repeat;
+    @include bidi(((padding-right, 32px, padding-left, 0),));
```

Comment:
Looks like RTL is missing some right margin value

<img width="168" alt="screenshot 2019-02-26 at 09 26 30" src="https://user-images.githubusercontent.com/400117/53402189-33a1aa80-39a9-11e9-8e94-45cedbad2de9.png">

Reply:
✅